guides/source/command_line.md

```diff
@@ -380,15 +380,15 @@ $ bin/rails destroy model Oops
       remove      test/fixtures/oops.yml
 ```
 
-bin/rails
+rails
 ---------
 
-Since Rails 5.0+ has rake commands built into the rails executable, `bin/rails` is the new default for running commands.
+Since Rails 5.0+ has rake commands built into the rails executable, `rails` is the new default for running commands.
```

Comment:
Okay, yeah.. this reads super strangely. In the middle of a list of `rails` subcommands, we stop and talk about a different set of `rails` subcommands.

If you're up for it, I think this deserves some more aggressive surgery, to better integrate previously-rake commands into the outer list (based on their individual predicted-frequency-of-use).

Reply:
Sure I am. I will let you both now when I am back with changes done. Thanks you both for your reviews and great hints!